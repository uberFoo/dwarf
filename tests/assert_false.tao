fn main() {
    chacha::assert_eq(false, true);
}
