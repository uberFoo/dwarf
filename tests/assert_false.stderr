Interpreter exited with:

[1;31merror[0m: assertion failed at [4;33mchacha::assert_eq(false, true)[0m.
  --> Found `false`, expected `true`.

